frontend/src/components/priorityClass/__snapshots__/priorityClassDetails.stories.storyshot

```diff
@@ -72,20 +72,7 @@ exports[`Storyshots PriorityClass/PriorityClassDetailsView Default 1`] = `
               />
               <div
                 class="MuiGrid-root MuiGrid-item css-13i4rnv-MuiGrid-root"
-              >
```

Comment:
I wonder why this button isn't rendered now?

Reply:
I think auth check didn't work before. Now, it falls back, I would say. I didn't found any test which is already testing auth.

Reply:
@illume Do you know how to test or have an example to test AuthVisible?

Reply:
now edit is see able.